src/shared/components/palette/collection/write-collection-field.component.ts

```diff
@@ -154,11 +148,13 @@ export class WriteCollectionFieldComponent extends AbstractFieldWriteComponent i
     if (this.isExpanded) {
       return false;
     }
-    let id = false;
-    if (this.formArray.at(index)) {
-      id = this.formArray.at(index).get('id').value;
+    const id = this.getControlIdAt(index);
+    if (!!id) {
+      if (!!this.profile.user && !!this.profile.user.idam) {
+        return !this.profile.user.idam.roles.find(role => this.hasUpdateAccess(role));
```

Comment:
Just wanted to check that `find()` will be transpiled correctly for IE11, since `find()` isn't supported for that browser

Reply:
This was already in place. We simply made sure there was a `user` and it had an appropriate `idam` property before trying to use that property.